src/config/containers-and-vms/index.md

```diff
@@ -0,0 +1,9 @@
+# Containers and Virtual Machines
+
+This section describes how to set up some of the container and virtual machine
+systems available on Void.
```

Comment:
"systems" or "tools" ?

Reply:
"Tools" feels odd to me here, perhaps too specific? E.g. i'd described 'grep' as a tool, but wouldn't describing (say) LibreOffice as an "office productivity tool". How about "software"?

Reply:
software sounds good!